app/src/main/java/com/iyanuoluwa/imovie/NowPlaying.kt

```diff
@@ -12,9 +12,9 @@ import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
-import com.iyanuoluwa.imovie.adapter.MovieAdapter
-import com.iyanuoluwa.imovie.api.MovieJson
-import com.iyanuoluwa.imovie.data.ApiNowPlaying
+import com.iyanuoluwa.imovie.ui.adapter.MovieAdapter
+import com.iyanuoluwa.imovie.data.model.buffer.api.MovieJson
+import com.iyanuoluwa.imovie.data.remote.request.ApiNowPlaying
```

Comment:
We'll figure out how to consolidate all your API calls into one interface, but for now, this ApiNowPlaying file should just be in the `remote` package directly. It's neither a request nor response.

```suggestion
import com.iyanuoluwa.imovie.data.remote.ApiNowPlaying
```